config/locales/en.yml

```diff
@@ -0,0 +1,56 @@
+en:
+  label_redmine_zulip: Zulip
+  zulip_settings_header: Zulip Plugin Configuration
+  zulip_settings_label_url: Zulip API URL
+  zulip_settings_label_email: Zulip Bot Email
+  zulip_settings_label_api_key: Zulip Bot API key
+  field_zulip_stream: Stream name
+  field_zulip_private_messages: Get private notifications
+  field_zulip_subject_issue: Create subjects for issues
+  field_zulip_subject_version: Create subjects for versions
+  zulip_notify_assignment: >
+    %{user} **assigned** issue [%{id}: %{subject}](%{url}) to **you**
+
+
+    ~~~quote
+
+    %{description}
+
+    ~~~
+
+
+    **%{status_label}**: %{status}
+
+
+  zulip_notify_unassignment: >
+    %{user} **unassigned** issue [%{id}: %{subject}](%{url}) from **you**
+
+  zulip_notify_updated: >
+    %{user} **updated** issue [%{id}: %{subject}](%{url})
+
+  zulip_notify_destroyed: >
+    %{user} **deleted** issue **%{id}: %{subject}**
+
+  zulip_init_issue_subject: >
+    %{user} **created** issue [%{id}: %{subject}](%{url})
+
+
+    ~~~quote
+
+    %{description}
+
+    ~~~
+
+    **%{assigned_to_label}**: %{assigned_to}
+
+    **%{status_label}**: %{status}
```

Comment:
We should probably use bullet points here! In our integrations, any time a ticket or issue has attributes we'd like to list, we just structure them as a bulleted list, it looks a lot cleaner when rendered!